### ISSUE TITLE
Fix accessibility issues

### ISSUE THREAD

**Issue by yaaax** [cnumr/EcoIndex#71]
Fix accessibility issues throughout the whole website

- the component giving the EcoIndex grade (A to G) is not accessible. Current letter element must use a `aria-current` attribute.
- …
